examples/tools/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_subdirectory(elf_run)
+add_subdirectory(elf_debug)
```

Comment:
I think it's a mistake since there is no ``elf_debug`` in the MR

Reply:
Indeed sorry about this... Fixed!